pom.xml

```diff
@@ -257,7 +257,7 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
-                <version>2.10.4</version>
+                <version>3.1.0</version>
```

Comment:
ℹ️ Maven Javadoc Plugin v3.2.0 was released in the meantime, apparently on March 12, 2020.

Reply:
Dependency has been upgraded to version 3.2.0